bin/shpec

```diff
@@ -21,9 +21,9 @@ describe() {
 
 end() {
   : $((test_indent -= 1))
-  if [ $test_indent -eq 0 ]; then
-    [ $failures -eq 0 ]
-  fi
+  [ $test_indent -ge 0 ] && return
+  printf -u 2 "Syntax Error in ${_shpec_file} file\n"
```

Comment:
Even in the original branch, `${_shpec_file}` is not defined here. If there's not an easy way to get the name of the file, I'd rather just omit that part for now.

Also, I was unable to find any documentation on the `-u` option for printf. What does it do, is it needed, and can we use our internal `echoe` instead?